### PR TITLE
row col-4 .dillan html/css add

### PR DESCRIPTION
**Pull request by teaglebuilt**
For some reason my container size was not the same height as yours. Im not quite sure why. could not find where the height and width setting was in css mixed with the css framework/col-4.

accidently added a couple files that i had to ignore. .firebasearc and ds store. 

If i did anything else wrong plz let me know.

### CODE REVIEW

**Review by davi3blu3 — APPROVED** [review]
Looks good! Approving everything except the changes to .firebaserc which I should have gitignored originally - that's unique to each of us.

I haven't seen the linear-gradient property used with additional styles after the parenthesis are closed? Not sure what that's doing exactly, but I'm curious to hear where you were going with that. Other than that your syntax looks good - nice job!

**Comment by teaglebuilt** [comment]
hey so the additional styles after the linear gradient quote are actually
for the linear gradient. The size of the linear-gradients pattern. If i
didnt give it a size it would just appear black.

I also updated my css and fixed the height so its equal to your container
and filled the portfolio and facebook link as well. Although when i sent it
to github it never showed up. i tried it again and the terminal said i had
basically already done it . See you at class
